client/client-benchmark-noop-api-plugin/src/main/java/org/elasticsearch/plugin/noop/action/search/TransportNoopSearchAction.java

```diff
@@ -40,7 +40,7 @@ protected void doExecute(Task task, SearchRequest request, ActionListener<Search
                 new SearchHit[0], new TotalHits(0L, TotalHits.Relation.EQUAL_TO), 0.0f),
             InternalAggregations.EMPTY,
             new Suggest(Collections.emptyList()),
-            new SearchProfileShardResults(Collections.emptyMap()), false, false, 1),
+            new SearchProfileResults(Collections.emptyMap()), false, false, 1),
```

Comment:
Sorry for the renames. The old names were confusing and I introduces a couple of new classes which would have been even more confusing without the rename.